### PR TITLE
Update FAQ outdated content and link

### PR DESCRIPTION
**Pull request by lilyndk**
## Description
FAQ content updated in yml file.

## Change Type
- [X] Bug Fix
- [ ] New Feature
- [ ] Code Refactor
- [ ] Mentor Update
- [ ] Documentation
- [X] Other

## Related Issue
[

](https://github.com/WomenCodingCommunity/WoenCodingCommunity.github.io/issues/72)

## Pull request checklist

Please check if your PR fulfills the following requirements:
- [X] I checked and followed the [contributor guide](CONTRIBUTING.md) 
- [X] I have tested my changes locally.
- [ ] I have added a screenshot from the website after I tested it locally

### CODE REVIEW

File: _data/faq.yml [hunk 1]
```diff
@@ -83,23 +82,22 @@
 - question: What are the expectations for mentees in the mentorship programme?
   answer: |
     The expectations for mentees may include setting and achieving goals, being open to feedback and guidance, actively seeking out opportunities to learn, and being committed to their own personal and professional growth. Mentees are expected to actively participate in the mentorship relationship, be open to learning and be willing to share their own experiences and knowledge. <br>
-    <a href="code-of-conduct.html">Please, find more information in Code of Conduct for Mentees</a>
+    <a href="/mentorship-code-of-conduct">Please, find more information in Code of Conduct for Mentees</a>
```
Comment (dricazenck) [inline _data/faq.yml]:
```suggestion
    <a href="/mentorship-code-of-conduct#mentee">Please, find more information in Code of Conduct for Mentees</a>
```

Could you add anchor for that and for the option above, please.

File: _data/faq.yml [hunk 2]
```diff
@@ -83,23 +82,22 @@
...
     It's important for both the mentor and mentee to have a clear understanding of the time commitment required for the mentorship programme. 
     They should also be able to set their own schedule and make adjustments as needed, as long as it is mutually agreed upon. 
     It is important for both parties to make the most of their time together and try to achieve the goals they set together. <br><br>
-    <a href="code-of-conduct.html">Please, find more information in Code of Conduct</a>
+    <a href="/mentorship-code-of-conduct">Please, find more information in Code of Conduct</a>
   index: 17
 
 - question: How long does a mentorship cycle last? 
-  answer: <p> The Mentorship programme lasts from March till November 2024, <a href="timeline.html"> More detail </a></p>
+  answer: <p> The Mentorship programme lasts from March till November 2024.</p>
```
Comment (dricazenck) [inline _data/faq.yml]:
```suggestion
  answer: <p> The Mentorship programme lasts from March till November of the same year.</p>
```

**Review by dricazenck — APPROVED** [review]

File: _data/faq.yml [hunk 3]
```diff
@@ -83,23 +82,22 @@
 - question: What are the expectations for mentees in the mentorship programme?
   answer: |
     The expectations for mentees may include setting and achieving goals, being open to feedback and guidance, actively seeking out opportunities to learn, and being committed to their own personal and professional growth. Mentees are expected to actively participate in the mentorship relationship, be open to learning and be willing to share their own experiences and knowledge. <br>
-    <a href="code-of-conduct.html">Please, find more information in Code of Conduct for Mentees</a>
+    <a href="/mentorship-code-of-conduct#mentee">Please, find more information in Code of Conduct for Mentees</a>
```
Comment (dricazenck) [inline _data/faq.yml]: You have to check if the id is correct in the code-of-conduct, please.

Comment (lilyndk) [reply _data/faq.yml]: noted :)

Comment (lilyndk) [reply _data/faq.yml]: Fixed in last commit

**Pull request merged**